src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1078,6 +1078,11 @@
       "run": "after",
       "snippetPath": "inventory-hierarchy/addEffectiveHoldingsToItemsAndHoldingsViewDuringUpgrade.sql",
       "fromModuleVersion": "21.0.3"
+    },
+    {
+      "run": "after",
+      "snippetPath": "populatePublicationPeriod.sql",
+      "fromModuleVersion": "21.0.7"
```

Comment:
Why is this 21.0.7 when it is to be first included in 21.0.9?

Won't this mean that if folks upgrade from 21.0.8 to 21.0.9 this won't run (because 21.0.7 has already passed)?

Reply:
Yes, good catch, thanks! Changed to 21.0.9.